executor.py

```diff
@@ -68,13 +71,14 @@ def index(self, docs: DocumentArray, **kwargs):
     def search(
         self,
         docs: 'DocumentArray',
+        parameters: Dict = {},
```

Comment:
maybe also document the parameter ?

Reply:
and also say what is the parameter filter and you can set it, where to get documentation about it, ...

Reply:
because these information in the docstring go to hub website and will be parsed for users

Reply:
Docstring of the Executor endpoint goes on hub ?
I should not precise the filter here as this is general parameters. It contains things from core as well like `runtine args` etcc.

Rather I documented how to use filter in the README.md https://github.com/jina-ai/executor-qdrant-indexer/tree/feat-pre-filtering#vector-search